torch/onnx/utils.py

```diff
@@ -332,6 +335,9 @@ def _export(model, args, f, export_params=True, verbose=False, training=False,
             opset_version = _default_onnx_opset_version
         _set_opset_version(opset_version)
         _set_operator_export_type(operator_export_type)
+        val_keep_init_as_ip = True if keep_initializers_as_inputs is None else keep_initializers_as_inputs
```

Comment:
Nit: why not just directly update `keep_initializers_as_inputs`?

Reply:
Not a very strong reason, but just that as a practice I keep input arguments as immutable, in case there's other parts of the code (in the past or future) that need it.